projects/canopy/src/styles/variables/_typography.scss

```diff
@@ -1,5 +1,6 @@
 :root {
   --font-family: 'Roboto', arial, Sans-Serif;
+  --font-family-expressive: 'Lyon', 'Roboto', arial, Sans-Serif;
```

Comment:
I wondered about the fallbacks on here, given Lyon is a serif font I wondered if the fallback should be `Serif` and something other than `Roboto`? Could be a design question.

Reply:
Ordinarily, I'd agree. However, Brand have made it clear that they don't want any serif alternatives to be used in place of Lyon. As such, probably best to fall back to Roboto as it has equivalent styles and is used on headings too.